composer.json

```diff
@@ -26,6 +26,7 @@
         "drupal/devel": "dev-8.x-1.x",
         "drupal/drupal-extension": "dev-assert-not-button as 3.1.6",
         "drupal/phing-drush-task": "1.0",
+        "drupal/search_api": "8.1.0-alpha12",
```

Comment:
I only put it in require-dev because this is a temporary solution. In the future, a requirement of the search_api_solr package will be included and this is going to download the search_api as well. I can move it but it is indeed meant for the development environment.